### PR TITLE
Recognize phone handle

### PR DESCRIPTION
**Pull request by ganmacs**
大体友達感ある

ref  #31

### CODE REVIEW

**Comment by ganmacs** [comment]
@Sushisushi-sandesu/leap-team :innocent:

File: Assets/Scripts/ScreenShotScript.cs [hunk 1]
```diff
@@ -13,13 +13,15 @@ public class ScreenShotScript : MonoBehaviour
...
 	private PeaceRecognizer pr;
+	private PhoneHandleRecognizer phr;
 
 	// Use this for initialization
 	void Start ()
 	{
 		screenShotTexture = new Texture2D (100, 100);
 		screenShot.texture = screenShotTexture;
-		pr = new PeaceRecognizer (TakeScreenShot);
+		//pr = new PeaceRecognizer (TakeScreenShot);
```
Comment (tomoasleep) [inline Assets/Scripts/ScreenShotScript.cs]: Delete this line.

**Comment by takuti** [comment]
ii-to-omou

**Comment by ganmacs** [comment]
arigatai

**Comment by tomoasleep** [comment]
(ごちうさ画像)

**Pull request merged**